src/Sylius/Component/Core/StateResolver/OrderPaymentStateResolver.php

```diff
@@ -94,7 +94,10 @@ private function getTargetTransition(OrderInterface $order): ?string
             $completedPaymentTotal += $payment->getAmount();
         }
 
-        if (0 < $completedPayments->count() && $completedPaymentTotal >= $order->getTotal()) {
+        if (
+            (0 < $completedPayments->count() && $completedPaymentTotal >= $order->getTotal()) ||
```

Comment:
Side note: I hate Yoda style for comparisons, zero is less than some variable -.-